### PR TITLE
Support display Image with https link in Juptyer

### PR DESCRIPTION
**Pull request by eddyxu**

### CODE REVIEW

**Comment by eddyxu** [comment]
Screen Shot 2021-02-18 at 7 19 22 PM

File: python/rikai/types/vision.py [hunk 1]
```diff
@@ -119,7 +116,8 @@ def display(self, **kwargs):
         """
         from IPython.display import Image
 
-        return Image(self.uri, **kwargs)
```
Comment (chunyang) [inline python/rikai/types/vision.py:122]: Curious why many libraries don't support loading from HTTPs out of the box (Jupyter, TensorFlow, others?). Jupyter appears to support http but not https?

**Review by chunyang — APPROVED** [review]
:shipit:

File: python/rikai/types/vision.py (hunk 1 above)
Comment (eddyxu) [reply python/rikai/types/vision.py:122]: Usually a good design should design against to a "file-object" like interface, i'd say.

**Pull request merged**